app/models/va_profile_redis/v2/contact_information.rb

```diff
@@ -179,6 +179,10 @@ def response_from_redis_or_service
       def contact_info_service
         @service ||= VAProfile::V2::ContactInformation::Service.new @user
       end
+
+      def verified_user?
+        @user.icn.present?
```

Comment:
User's with ICN are verified users. Only verified users are permitted to profile services.